packages/frontend/src/app/[locale]/claims/page.tsx

```diff
@@ -0,0 +1,13 @@
+import { Claims } from "@/src/components/claims";
+import { Nav } from "@/src/components/nav";
+
+import styles from "../styles.module.css";
+
+export default function CampaignsCreatePage() {
```

Comment:
Wrong component name.

Reply:
Addressed in 71253e78b54d3ab9d43fe6cbb62796d87a92e66b